server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QualityGatesWsSupport.java

```diff
@@ -68,12 +89,37 @@ boolean isQualityGateAdmin() {
       .build();
   }
 
+  OrganizationDto getOrganization(DbSession dbSession, Request request) {
+    String organizationKey = Optional.ofNullable(request.param(PARAM_ORGANIZATION))
+      .orElseGet(() -> defaultOrganizationProvider.get().getKey());
+    Optional<OrganizationDto> organizationDto = dbClient.organizationDao().selectByKey(dbSession, organizationKey);
+    return checkFoundWithOptional(organizationDto, "No organization with key '%s'", organizationKey);
+  }
+
+  /**
+   * @deprecated use {@link #checkCanEdit(QGateWithOrgDto)} instead
+   */
+  @Deprecated
   void checkCanEdit(QualityGateDto qualityGate) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)